src/Razor/src/Microsoft.VisualStudio.Editor.Razor/TelemetryReporter.cs

```diff
@@ -1,318 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
This class became `VSTelemetryReporter`, and mutual logic between it and `DevKitTelemetryReporter` was moved to an abstract class called `TelemetryReporter`